packages/core/src/credential/Credential.ts

```diff
@@ -293,27 +285,109 @@ type VerifyOptions = {
 
 /**
  * Verifies data structure & data integrity of a credential object.
+ * This combines all offline sanity checks that can be performed on an ICredential object.
+ * A credential is valid only if it is well formed AND there is an on-chain attestation record referencing its root hash.
+ * To check the latter condition as well, you need to call [[verifyCredential]] or [[verifyPresentation]].
  *
  * @param credential - The object to check.
  * @param options - Additional parameter for more verification steps.
  * @param options.ctype - CType which the included claim should be checked against.
  */
-export async function verifyCredential(
+export function verifyWellFormed(
   credential: ICredential,
   { ctype }: VerifyOptions = {}
-): Promise<void> {
+): void {
   verifyDataStructure(credential)
   verifyDataIntegrity(credential)
 
   if (ctype) {
-    verifyAgainstCType(credential, ctype)
+    verifyClaimAgainstSchema(credential.claim.contents, ctype)
   }
 }
 
 /**
- * Verifies data structure, data integrity and the claimer's signature of a credential presentation.
+ * Queries the attestation record for a credential and matches their data. Fails if no attestation exists, if it is revoked, or if the attestation data does not match the credential.
```

Comment:
> as well as info on which trust policy led to acceptance of the credential.

Where?

Reply:
I think you reviewed an old version? I recognise the quote but I changed that afaik. At least it's not in the line you commented on?

Reply:
Shit ok, no idea what happened to my VS Code.